app/views/shared/_post.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="user-posts">
+  <h2>Post #<%= post.id %></h2>
+  <p><%= truncate(post.text, length: 110, omission: " . . .") %></p>
+  <div class="like-and-comment">
+    <p>Comments: <%= post.comment_counter %>, Likes: <%= post.like_counter %></p> 
```

Comment:
- [ ] When your app starts then it shows no count when there are no comments or likes because you are using the logic here which will get the comments and likes counts if there would be likes or comments now show null here kindly make your line 6 like below :arrow_down: :blush:

Just add `|| 0` after each counter so if these counters return null value then your view will show zero `0` :+1:

`<p>Comments: <%= post.comment_counter || 0 %>, Likes: <%= post.like_counter || 0 %></p> `